aptos/typescript/backend_examples/src/invisible_wallet.ts

```diff
@@ -0,0 +1,88 @@
+// 1. Import everything we need for the tutorial
+import { 
+    AccountAddress,
+    AccountAuthenticatorEd25519,
+    Aptos, 
+    AptosConfig, 
+    Network, 
+    SimpleTransaction 
+} from "@aptos-labs/ts-sdk";
+import { 
+    KeyClient, 
+    WalletClient, 
+    ShinamiWalletSigner  
+} from "@shinami/clients/aptos";
+
+// 2. Copy your access key value
+const ALL_SERVICES_TESTNET_ACCESS_KEY = "{{allServicesTestnetAccessKey}}";
+
+// 3. Set up a walletId an its associated secret. Just for the tutorial. Your
+//    app should figure out the best way to manage it's wallet Ids and secrets.
+const WALLET_ID = "{{walletID}}";
+const WALLET_SECRET = "{{walletSecret}}";
+
+// 4. Instantiate your Aptos and Shinami clients
+const aptosClient = new Aptos(new AptosConfig({ network: Network.TESTNET}));
+const keyClient = new KeyClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const walletClient = new WalletClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+
+// 5. Create a ShinamiWalletSinger to more easily manage the Invisible Wallet
+const signer = new ShinamiWalletSigner(
+    WALLET_ID,
+    walletClient,
+    WALLET_SECRET,
+    keyClient
+);
+
+// 6. Create the Invisible Wallet
+const CREATE_WALLET_IF_NOT_FOUND = true;
+const INITIALIZE_ON_CHAIN = false;
+const walletAddress = await signer.getAddress(CREATE_WALLET_IF_NOT_FOUND, INITIALIZE_ON_CHAIN);
+
+// 7. Sponsor and execute a transaction for the Invisible Wallet
+const simpleTx = await simpleMoveCallTransaction(walletAddress); 
+const pendingTx = await signer.executeGaslessTransaction(simpleTx);
```

Comment:
suggestion: You could call out that the fund associated with the key used here requires funds for this to succeed

Reply:
good call. Added